src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetProjectManagerService.cs

```diff
@@ -533,7 +533,7 @@ private static string CreateProjectActionId()
                 else
                 {
                     throw new ArgumentException(
-                        string.Format(Strings.ProjectWithIdNotFound, CultureInfo.CurrentCulture, projectId),
+                        string.Format(CultureInfo.CurrentCulture, Strings.ProjectWithIdNotFound, projectId),
```

Comment:
This is a secondary fix.  The test for it is [here](https://github.com/NuGet/NuGet.Client/pull/3645/files#diff-0c1b01280c4ac523b1c8e3d6fe76cad9R74).